pkg/infra/tracing/tracing.go

```diff
@@ -99,36 +99,19 @@ func ProvideService(tracingCfg *TracingConfig) (*TracingService, error) {
 	return ots, nil
 }
 
-func (ots *TracingService) GetTracerProvider() tracerProvider {
-	return ots.tracerProvider
-}
-
-func TraceIDFromContext(ctx context.Context, requireSampled bool) string {
-	spanCtx := trace.SpanContextFromContext(ctx)
-	if !spanCtx.HasTraceID() || !spanCtx.IsValid() || (requireSampled && !spanCtx.IsSampled()) {
-		return ""
-	}
-
-	return spanCtx.TraceID().String()
-}
-
-// Error sets the status to error and record the error as an exception in the provided span.
-func Error(span trace.Span, err error) error {
-	attr := []attribute.KeyValue{}
-	grafanaErr := errutil.Error{}
-	if errors.As(err, &grafanaErr) {
-		attr = append(attr, attribute.String("message_id", grafanaErr.MessageID))
-	}
-
-	span.SetStatus(codes.Error, err.Error())
-	span.RecordError(err, trace.WithAttributes(attr...))
-	return err
+func NewNoopTracerService() *TracingService {
+	tp := &noopTracerProvider{TracerProvider: noop.NewTracerProvider()}
+	otel.SetTracerProvider(tp)
+
+	cfg := NewEmptyTracingConfig()
+	cfg.Propagation = "jaeger,w3c"
```

Comment:
why is propagation enabled when tracing is disabled?

Reply:
Very good question 😂

Reply:
I'll remove it